nservicebus/upgrades/5to6/container.md

```diff
@@ -54,4 +54,14 @@ snippet: 5to6-ExplicitProperties
 
 To access it at runtime create a new [`Feature`](/nservicebus/pipeline/features.md) and put the following code in the `.Setup` method
 
-snippet: 5to6-IConfigureComponentsNotInjected
\ No newline at end of file
+snippet: 5to6-IConfigureComponentsNotInjected
+
+## Existing passed to the configuration API are no longer disposed
+
+snippet:Autofac_Existing
+
+Above snippet shows an example with Autofac how an existing container can be passed into the configuration API but the described behavior applies to all container adapters which are [officially supported](/nservicebus/containers/). Previous versions of the containers treated the externally passed in container as if it was owned by the container adapter and disposed the container when the bus was disposed. This behavior changed. When a container customization is passed like above, then the container is no longer disposed. It is the responsibility of the container owner to dispose the container. 
```

Comment:
>Above snippet shows an example with Autofac how an existing container can be passed into the configuration API but the described behavior applies to all container adapters which are [officially supported](/nservicebus/containers/). 

While the above example shows how an existing container can be passed into the configuration API using Autofac, the same behavior can also be applied to all of the [currently supported container adapters](/nservicebus/containers/).